modules/transactionpool/accept.go

```diff
@@ -311,6 +311,15 @@ func (tp *TransactionPool) acceptTransactionSet(ts []types.Transaction, txnFn fu
 	tp.transactionSetDiffs[setID] = cc
 	tsetSize := len(encoding.Marshal(ts))
 	tp.transactionListSize += tsetSize
+	blockHeight, err := tp.getBlockHeight(tp.dbTx)
```

Comment:
we should probably keep this value in memory. Let's extend the putBlockHeight function to also set the in-memory block height in the tpool to the same value, and then read that instead of going to disk.